.github/workflows/changesets.yml

```
@@ -0,0 +1,39 @@
+name: Release
```

```suggestion
name: Release NPM Packages
```

I think it's nice to make this more explicit what this action is doing.